### PR TITLE
Feature/151168137/gaming screen

### PR DESCRIPTION
**Pull request by segunolalive**
#### What does this PR do?
Redesigns the gaming screen

#### Description of Task to be completed?
Re-Design UI for the actual gaming screen

#### How should this be manually tested?
npm start

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#151168137

#### Screenshots (if appropriate)
localhost-3000- 1
localhost-3000- iphone 6 plus

### CODE REVIEW

File: app/controllers/answers.js [hunk 1]
```diff
@@ -8,6 +8,23 @@ const _ = require('underscore');
...
+ * @param {Object}   res  express http response object
+ * @param {Function} next calls the next function in the middleware stack
+ * @return {mixed}        sends an http response or calls the next middleware
+ */
+exports.add = (req, res, next) => {
+  const answer = new Answer(req.body);
+  answer.save()
+    .then(() => res.send({
+      success: true,
+      message: 'Question added successfully',
```
Comment (nebanat) [inline app/controllers/answers.js]: Please change message to 'Answer added successfully'.

**Review by nebanat — CHANGES_REQUESTED** [review]
Please make the appropriate change

File: config/routes.js [hunk 2]
```diff
@@ -82,15 +82,13 @@ module.exports = (app, passport, auth) => {
   app.param('userId', users.user);
 
   // Answer Routes
-  app.post('/api/answers/add', answers.add);
```
Comment (dinobi) [inline config/routes.js]: This route was removed, any reason for that?

File: config/routes.js [hunk 3]
```diff
@@ -82,15 +82,13 @@ module.exports = (app, passport, auth) => {
...
   app.get('/answers', answers.all);
   app.get('/answers/:answerId', answers.show);
   app.get('/answers', answers.all);
   // Finish with setting up the answerId param
   app.param('answerId', answers.answer);
 
   // Question Routes
-  app.post('/api/questions/add', questions.add);
```
Comment (dinobi) [inline config/routes.js]: This route was removed, any reason for that

**Review by nebanat — APPROVED** [review]

**Comment by ibonly** [comment]
Why was this PR merged without my approval?

**Comment by segunolalive** [comment]
I'm sorry. This merge was a mistake. It shouldn't have been merged yet

**Pull request merged**